install.sh

```diff
@@ -994,11 +994,26 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
-- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
-    (echo; echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"') >> ${shell_profile}
-    eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"
+# show different instructions on adding Homebrew to path based on if the user has already done so
+if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}"
+then
+  if ! type brew &>/dev/null
+  then
+    cat <<EOS
+    - Run this command in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
+      eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"
+EOS
+  else
```

Comment:
Why the `else` not doing anything? Having an `else` is not mandatory.

Reply:
I know. I included it because it made the code look better than just having an if statement without the else. Shall I remove it?